### DISCUSSION TITLE
The 'Brave browser' seems to render TailwindCSS text and size attributes much larger than chrome, edge and safari...

### DISCUSSION THREAD

**Discussion by marksmithjobapps** [tailwindlabs/tailwindcss#10706] [Help] [answered]
I have no idea why, but it seems that the Brave browser which I think uses the same underlying technology as Chrome/Edge and Safari seems to render Tailwincss text and size attributes (h- w-h etc.) larger than the other browsers - has anyone else run into this issue and is there a workaround or a fix other than not using the Brave browser? 

I have checked the same site I am working on in Chrome, Edge, Safari, Firefox, Opera and DuckDukGo browsers and they all render sized attributes 'correctly' and at the same size for the same viewing screen size, but the Brave browser seems to render things +/- 10% or so larger for some strange reason...

Any advice, pointers, workarounds or solutions will be much appreciated!

**Comment by wongjn** [comment, marked as answer]
By default, font sizes, heights, widths, etc. use `rem` unit which is defined as the <q>font size of the root element</q>[^1], where root element would be the `<html>` element. The out-of-the-box default is usually `16px` but it can be changed by browser settings. Perhaps look in the settings for your Brave browser instance and set it back to the default font size if it is not already the default.

[^1]: https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#relative_length_units

**Reply by marksmithjobapps** [reply]
Sir, you are a superstar! (and I am an idiot for not checking the browser settings)

I must have changed the font settings to large (from the recommended **medium**) in braves settings a while back and totally forgot that I had done that and when is saw the 'issue' I posted, I just assumed that the browser settings were at their default...

**Schoolboy error - always check your settings before posting :-)**

Thank you
